lib/Graph/Graph.cpp

```diff
@@ -276,14 +276,18 @@ class AbstractDottyPrinter {
     os << "}";
   }
 
-  void dumpNode(Node *N) {
+  void dumpNode(Node *N, bool uniqueNames) {
     if (!N) {
       return;
     }
     std::ostringstream os;
-    // Print a node descriptor that looks like this:
-    // vNNNN [ shape = "record" label = "{...}" ];
-    os << uniqueVertexName(N) << "[\n";
+    if (uniqueNames) {
+      // Print a node descriptor that looks like this:
```

Comment:
Think this comment should be bumped back above the if block here.

Reply:
The node descriptor doesn't look like that though, would you like me to add a comment with how the other node descriptor looks?

Reply:
Oh -- well I think the original comment was referring to everything being printed, not just the initial vertex name. NBD.

Reply:
I'll clean this up.